person.rb

```diff
@@ -1,11 +1,14 @@
 require_relative 'nameable'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
 
 class Person < Nameable
-  # Accessors Methods That used for getters and Setters in quick way!
+  # Accessors Methods That used for getters and Setters in a quick way!
   # attr_accessor works for getter and setter both purposes.
   attr_accessor :name, :age
```

Comment:
> - [ ] Great job on your implementation! However, I noticed that you missed including the `:rentals` attribute in the attr_accessor line. To rectify this, please update the attr_accessor line to include `:rentals`. This will ensure that you have proper access to the `:rentals` attribute in your Person class.👍